wheel_radius: 0.10
distance_wheels: 0.45
gear_ratio: 1.0

endcoder_resolution: 4096
pole_number: 10
can_hz: 10.0
max_rpm: 200
break_option: 1
rated_current: 8.0
max_current: 16.0
hall_offset: 240

can_device: can0
id: 1537

# synchronous: 1 |  asynchronous: 2
mode: 1
